lib/loaders/css-extract.js

```diff
@@ -22,13 +23,15 @@ module.exports = {
      */
     prependLoaders(webpackConfig, loaders) {
         if (!webpackConfig.extractCss) {
+
+            const options = {};
+
             // If the CSS extraction is disabled, use the
             // style-loader instead.
             return [{
                 loader: 'style-loader',
-                options: {
-                    sourceMap: webpackConfig.useSourceMaps,
```

Comment:
Doesn't our version still requires the `sourceMap` option?
It was removed in `style-loader@1.0.0` and until #710 is merged we're using `style-loader@^0.21.0`.

Reply:
I have included sourcemaps in the default options